repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py

```diff
@@ -71,11 +70,19 @@ def update_rhel_subscription(self, overlayfs_info):
             # + would be problem later). Needs a little investigation here about the impact, even
             # + in the container.
             for cert in existing_certs:
-                error = preparetransaction.check_container_call(overlayfs_info, ['rm', '-f', cert])
+                # FIXME: fails on insufficient permissions
+                cmd = ['rm', '-f', cert]
+                _, error = preparetransaction.guard_container_call(overlayfs_info, cmd)
```

Comment:
please don't use _ eventually this will be used for localization, use _unused, the underscore infront of it will avoid warnings about unused variables and

Reply:
fixed